### ISSUE TITLE
electrum.interface.RequestCorrupted: required field 'fee' missing from dict

### ISSUE THREAD

**Issue by dmp1ce** [romanz/electrs#291]
When I connect to my own Electrs server, Electrum is returning an error and keeps trying to get the information again over and over. I didn't see any errors from Electrs logs. Connecting to public nodes had no problem.

The error occured when I tried to send a transaction. Once the transaction became confirmed, the error stopped.

Electrs version 0.8.3
Electrum version 4.0.2

Electrum error:

```
E/i | interface.[xxxxxxxxx.onion:60002] | Exception in wrapper_func: RequestCorrupted("required field 'fee' missing from dict")
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/electrum/util.py", line 1037, in wrapper
    return await func(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/electrum/interface.py", line 466, in wrapper_func
    return await func(self, *args, **kwargs)
  File "/usr/lib/python3.8/site-packages/electrum/interface.py", line 490, in run
    await self.open_session(ssl_context)
  File "/usr/lib/python3.8/site-packages/electrum/interface.py", line 630, in open_session
    await group.spawn(self.monitor_connection)
  File "/usr/lib/python3.8/site-packages/aiorpcx/curio.py", line 242, in __aexit__
    await self.join()
  File "/usr/lib/python3.8/site-packages/aiorpcx/curio.py", line 211, in join
    raise task.exception()
  File "/usr/lib/python3.8/site-packages/electrum/synchronizer.py", line 82, in _start_tasks
    await group.spawn(self.main())
  File "/usr/lib/python3.8/site-packages/aiorpcx/curio.py", line 242, in __aexit__
    await self.join()
  File "/usr/lib/python3.8/site-packages/aiorpcx/curio.py", line 211, in join
    raise task.exception()
  File "/usr/lib/python3.8/site-packages/electrum/synchronizer.py", line 171, in _on_address_status
    result = await self.interface.get_history_for_scripthash(h)
  File "/usr/lib/python3.8/site-packages/electrum/interface.py", line 922, in get_history_for_scripthash
    assert_dict_contains_field(tx_item, field_name='fee')
  File "/usr/lib/python3.8/site-packages/electrum/interface.py", line 118, in assert_dict_contains_field
    raise RequestCorrupted(f'required field {field_name!r} missing from dict')
electrum.interface.RequestCorrupted: required field 'fee' missing from dict
```

Electrs log:

```
electrs_1      | 2020-08-10T00:57:19.858+00:00 - DEBUG - [127.0.0.1:58804] shutting down connection
electrs_1      | 2020-08-10T00:57:19.860+00:00 - INFO - [127.0.0.1:58804] disconnected peer #341
electrs_1      | 2020-08-10T00:57:21.297+00:00 - INFO - [127.0.0.1:58808] connected peer #342
electrs_1      | 2020-08-10T00:57:21.435+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.version\", \"id\": 0, \"params\": [\"electrum/4.0.2\", \"1.4\"]}\n")
electrs_1      | 2020-08-10T00:57:21.967+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.headers.subscribe\", \"id\": 1}\n")
electrs_1      | 2020-08-10T00:57:22.057+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 2, \"params\": [25]}\n")
electrs_1      | 2020-08-10T00:57:22.060+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 3, \"params\": [10]}\n")
electrs_1      | 2020-08-10T00:57:22.062+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 4, \"params\": [5]}\n")
electrs_1      | 2020-08-10T00:57:22.062+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 5, \"params\": [2]}\n")
electrs_1      | 2020-08-10T00:57:22.392+00:00 - DEBUG - failed to send PeriodicUpdate to peer 4: sending on a full channel
electrs_1      | 2020-08-10T00:57:22.393+00:00 - DEBUG - failed to send PeriodicUpdate to peer 7: sending on a full channel
electrs_1      | 2020-08-10T00:57:22.394+00:00 - TRACE - RPC PeriodicUpdate
electrs_1      | 2020-08-10T00:57:22.448+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.banner\", \"id\": 6}\n")
electrs_1      | 2020-08-10T00:57:22.452+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.donation_address\", \"id\": 7}\n")
electrs_1      | 2020-08-10T00:57:22.454+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.peers.subscribe\", \"id\": 8}\n")
electrs_1      | 2020-08-10T00:57:22.455+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.relayfee\", \"id\": 9}\n")
electrs_1      | 2020-08-10T00:57:22.456+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"mempool.get_fee_histogram\", \"id\": 10}\n")
electrs_1      | 2020-08-10T00:57:22.457+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 11, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.459+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 12, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.466+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 13, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.468+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 14, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.469+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 15, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.471+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 16, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.472+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 17, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.474+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 18, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.476+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 19, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.477+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 20, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.479+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 21, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.480+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 22, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.481+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 23, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.482+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 24, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.482+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 25, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.536+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 26, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.551+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 27, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.553+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 28, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.556+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 29, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.557+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 30, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.558+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 31, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.559+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 32, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.561+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 33, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.562+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 34, \"params\": [\"xxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.571+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 35, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.655+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 36, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.658+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 37, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.660+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 38, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.662+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 39, \"params\": [\"xxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.664+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 40, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:22.666+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 41, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:23.195+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.get_history\", \"id\": 42, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:23.734+00:00 - TRACE - RPC Done
electrs_1      | 2020-08-10T00:57:23.736+00:00 - DEBUG - [127.0.0.1:58808] shutting down connection
electrs_1      | 2020-08-10T00:57:23.738+00:00 - INFO - [127.0.0.1:58808] disconnected peer #342
electrs_1      | 2020-08-10T00:57:25.226+00:00 - INFO - [127.0.0.1:58812] connected peer #343
electrs_1      | 2020-08-10T00:57:25.317+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.version\", \"id\": 0, \"params\": [\"electrum/4.0.2\", \"1.4\"]}\n")
electrs_1      | 2020-08-10T00:57:25.795+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.headers.subscribe\", \"id\": 1}\n")
electrs_1      | 2020-08-10T00:57:25.886+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 2, \"params\": [25]}\n")
electrs_1      | 2020-08-10T00:57:25.887+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 3, \"params\": [10]}\n")
electrs_1      | 2020-08-10T00:57:25.888+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 4, \"params\": [5]}\n")
electrs_1      | 2020-08-10T00:57:25.889+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.estimatefee\", \"id\": 5, \"params\": [2]}\n")
electrs_1      | 2020-08-10T00:57:26.396+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.banner\", \"id\": 6}\n")
electrs_1      | 2020-08-10T00:57:26.399+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.donation_address\", \"id\": 7}\n")
electrs_1      | 2020-08-10T00:57:26.400+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"server.peers.subscribe\", \"id\": 8}\n")
electrs_1      | 2020-08-10T00:57:26.401+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.relayfee\", \"id\": 9}\n")
electrs_1      | 2020-08-10T00:57:26.402+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"mempool.get_fee_histogram\", \"id\": 10}\n")
electrs_1      | 2020-08-10T00:57:26.403+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 11, \"params\": [\"xxxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.403+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 12, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.403+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 13, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.417+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 14, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.419+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 15, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.420+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 16, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.421+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 17, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.422+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 18, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.423+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 19, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.424+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 20, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.470+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 21, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.471+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 22, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.471+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 23, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.472+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 24, \"params\": [\"xxxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.486+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 25, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.487+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 26, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.489+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 27, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.490+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 28, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.491+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 29, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.491+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 30, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.491+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 31, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.507+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 32, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.508+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 33, \"params\": [\"xxxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.509+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 34, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.510+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 35, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.511+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 36, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.520+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 37, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.521+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 38, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.522+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 39, \"params\": [\"xxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.537+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 40, \"params\": [\"xxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:26.549+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.subscribe\", \"id\": 41, \"params\": [\"xxxxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:27.048+00:00 - TRACE - RPC Request("{\"jsonrpc\": \"2.0\", \"method\": \"blockchain.scripthash.get_history\", \"id\": 42, \"params\": [\"xxxxxxx\"]}\n")
electrs_1      | 2020-08-10T00:57:27.399+00:00 - DEBUG - failed to send PeriodicUpdate to peer 4: sending on a full channel
electrs_1      | 2020-08-10T00:57:27.399+00:00 - DEBUG - failed to send PeriodicUpdate to peer 7: sending on a full channel
electrs_1      | 2020-08-10T00:57:27.400+00:00 - TRACE - RPC PeriodicUpdate
electrs_1      | 2020-08-10T00:57:27.510+00:00 - TRACE - RPC Done
electrs_1      | 2020-08-10T00:57:27.511+00:00 - DEBUG - [127.0.0.1:58812] shutting down connection
electrs_1      | 2020-08-10T00:57:27.511+00:00 - INFO - [127.0.0.1:58812] disconnected peer #343
electrs_1      | 2020-08-10T00:57:28.829+00:00 - INFO - [127.0.0.1:58816] connected peer #344
```

**Comment by Kixunil** [comment]
Oh shit, you've just leaked all your addresses to the internet - scripthash parameters. I suggest deleting this whole issue ASAP (is that even possible on GitHub?).

**Comment by dmp1ce** [comment]
Thanks! I `xxx` them out. Should I be worried? These are testnet coins?

**Comment by Kixunil** [comment]
Ah, testnet coins are fine even though it's still visible due to GitHub allowing to view edit history.

**Comment by dmp1ce** [comment]
I see. I'll just make a new issue.

**Comment by dmp1ce** [comment]
I deleted the revision history. Thanks again.

**Comment by Kixunil** [comment]
Ha, didn't know it's possible, good to know!

**Comment by dmp1ce** [comment]
Maybe giving a warning in the New Issue template for people posting logs would be a good idea.

**Comment by Kixunil** [comment]
I was also thinking about avoiding logging private information by default.

**Comment by romanz** [comment]
Could you please try electrs 0.8.5?
It should support Electrum 4 via https://github.com/romanz/electrs/commit/e55381fa6643661c0dad2c454415ed997084a13d.

**Comment by dmp1ce** [comment]
I'll try. I was waiting on 0.8.6 to upgrade because 0.8.4 and 0.8.5 didn't build due to outdated Dockerfile.

**Comment by romanz** [comment]
I can release a new version - but could you please check that the latest `master` is building correctly with Docker?

**Comment by dmp1ce** [comment]
Yep, I finally updated to master and the transactions work fine. No connection resets. Thank you!
